Documentation/Patterns.md

```diff
@@ -303,23 +304,23 @@ static StringView example_fn(const SourceLocation& loc = SourceLocation::current
     return loc.function_name();
 }
 
-int main(int, char**)
+int main(void)
```

Comment:
Undo this change. (writing an explicit `void` in a parameter list is C, where functions without any parameters are implicitly varargs. In C++, it's `main()`, but the LHS is fine too. The RHS is wrong.)